src/include/simeng/CoreInstance.hh

```diff
@@ -139,6 +137,10 @@ class CoreInstance {
 
   /** Reference to the global memory pointer */
   std::shared_ptr<simeng::memory::Mem> memory_ = nullptr;
+
+  /** Function, passed to the Core class, to communicate a syscall generated by
```

Comment:
- Would it be better to say: Callback function passed to the Core class in order to communicate a syscall generated by the Core's exception handler to SimOS' syscall handler?

Reply:
Reworded slightly but the use of SimOS is problematic as discussed https://github.com/UoB-HPC/SimEng/pull/285#discussion_r1114223608